frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -27,6 +29,16 @@ const ProjectDetails: React.FC = () => {
   const modelServingEnabled = featureFlagEnabled(
     dashboardConfig.spec.dashboardConfig.disableModelServing,
   );
+  const queryParams = useQueryParams();
+  const notebookLogout = queryParams.get('notebook_logout');
+  const notification = useNotification();
+
+  // TODO: this will be triggered twice, don't know why
+  React.useEffect(() => {
+    if (notebookLogout) {
+      notification.success(`Logout workbench successfully`);
```

Comment:
RE: my other comment

Remove the search param from the URL here. 
Docs: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/delete